pandas/tests/io/test_parquet.py

```diff
@@ -252,6 +301,10 @@ def test_duplicate_columns(self, fp):
 
     def test_unsupported(self, fp):
 
+        # bool with none
+        df = pd.DataFrame({'a': [True, None, False]})
```

Comment:
cc @martindurant this didn't pass (boolean with none) which is of ``object`` dtype.